CMakeLists.txt

```diff
@@ -1,48 +1,10 @@
 project (widelands)
 
-cmake_minimum_required (VERSION 2.8.7)
-cmake_policy(VERSION 2.8.7)
+cmake_minimum_required (VERSION 3.9)
+cmake_policy(VERSION 3.9)
 
 include("${CMAKE_SOURCE_DIR}/cmake/UseSystemInfo.cmake")
 
-# This policy is not known to versions prior 3.1 and would result in errors,
-# if set on such systems. This can be removed when cmake_minimum_required is set
-# to 3.1 or newer by using:
-# cmake_policy(VERSION 3.1)
-if(POLICY CMP0054)
-   cmake_policy(SET CMP0054 NEW)
-endif(POLICY CMP0054)
-
-# This policy is not known to versions prior 3.11 and would result in errors,
```

Comment:
well from the explanation in the text I would have expected we would need this on Trusty probably as we only require 3.9.
However as it compiles it could be good to go as well

Reply:
I get this on my trusty VM
 
```
CMake Error at CMakeLists.txt:3 (cmake_minimum_required):
  CMake 3.9 or higher is required.  You are running version 2.8.12.2
```

Reply:
Yes, i was hoping to use at least 3.12, but the Trusty images on Travis have only 3.9 available.
I see no errors in the logs, so the policy referring to be needed <3.11 seems to be no longer necessary (maybe due to updated cmake package files..)

@gunchleoc : Do you have "cmake3" as command available?

Reply:
There is a cmake3 package available for Trusty. When I install it, I get:

```
CMake Error at CMakeLists.txt:3 (cmake_minimum_required):
  CMake 3.9 or higher is required.  You are running version 3.5.1
```

My VM is a default installation + some installed dev packages, so that's the kind of setup we should be aiming for.

Reply:
Ah, interesting. I'll try to figure out why on Travis the Trusty comes with 3.9.
Would lowering our requirement to 3.5 ok? Or do you insist on staying compat to 2.8.x ?

[And with Trusty outlived its Standard Support Live time in April '19 maybe we should move to Xenial as lowest requirement / supported Distro.]

Reply:
3.5 is fine - it's available out of the box from the package manager.

I think we should try to still support trusty for a while, unless it becomes too ugly. Launchpad PPAs still support it.